TESTING.md

```diff
@@ -531,3 +532,14 @@ Repeat the above starting from different themes.
 #### Expected behaviour
 
 Changing theme should lead to full repaint of the colours according to the current theme.
+
+
+### Claude 3 Enterprise Testing
+#### Cody Gateway Instance
+- [ ] Ask Cody a question in a new chat window. The question should include a request for Cody to generate code with the site-admin configuration is set to **Claude 3 Sonnet** for chatModel and **Claude 3 Haiku** for completionModel
+- [ ] Ask Cody a question in a new chat window. The question should include a request for Cody to generate code with the site-admin configuration is set to **Claude 3 Opus** for chatModel and **Claude 3 Haiku** for completionModel
+- [ ] Generate a completion using the manual-trigger key binding on any primary languages (Javascript, Typescript, TypescriptReact, Python, Go).
+
+#### AWS Bedrock Instance
```

Comment:
Those steps are the same as **Cody Gateway Instance** testing steps.

Reply:
@pkukielka what's the best approach here? Should we be like "in each of these logins: ..., verify the following: ..."? Duplicate the steps? (We may want them to diverge soon enough anyway...)

Reply:
Ah this is where it gets tricky, so admins can choose either sonnet OR opus as their default model 🤔 We don't have a way for the QA team to switch configurations, SG02 is defaulted to sonnet at the moment

Reply:
Simply created two subcases for Enterprise accounts, one for each instance provided. I have also removed the model validation as the team doesn't have the configuration, and will simply test against whatever configuration is on the instance